miqcli/cli/main.py

```diff
@@ -1,7 +1,11 @@
 from copy import copy
 from functools import wraps
 from importlib import import_module
-from xmlrpclib import ServerProxy
+
+try:
+    from xmlrpclib import ServerProxy
+except ImportError:
+    from xmlrpc.client import ServerProxy
```

Comment:
We will probably have to handle few other incompatibilities between python versions. Wouldn't be better to use a _compat module within the main package to handle this? My suggestion, but I would be fine with this for now too.

Reply:
I am in favor also of having a compat module to handle incompatibilities between python versions. I think once more implementation is in place for the CLI, we can add that module in. Which we could then update the cli.main module to remove this handling of imports.